docker-start.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+[ -z "$UID" ] && UID=0
+[ -z "$GID" ] && GID=0
+
+# echo >> /etc/xxx and not adduser/addgroup because adduser/addgroup won't work if uid/gid already exists
+echo -e "xuser:x:${UID}:${GID}:xuser:/home/xuser:/bin/ash\n" >> /etc/passwd
+echo -e "xgroup:x:${GID}:xuser\n" >> /etc/group
+
+# it's better to do that (mkdir and chown) here than in the Dockerfile because it will be executed even on volumes if mounted
+mkdir -p /droppy-data 
+mkdir -p /droppy-data/config
+mkdir -p /droppy-data/files
```

Comment:
These directories changed recently (in 4.0):

`/droppy-data/config` is now `/config`
`/droppy-data/files` is now `/files`

If you can, please update. Otherwise, I'll do it once I test and land this.